schema/me/conversation/message.js

```diff
@@ -74,6 +74,9 @@ export const MessageType = new GraphQLObjectType({
     body: {
       description: "Text which has been parsed/sanitized of quoted replies (among other things).",
       type: GraphQLString,
+      resolve: ({ body }) => {
+        return body.split("\n\nAbout this collector:")[0]
+      },
```

Comment:
I believe that right now this field could be `null` when we _send_ a message because [our optimistic response](https://github.com/artsy/metaphysics/blob/4e0bfcc8eb90a62a34e517f1f89728c55c8590e9/schema/me/conversation/send_message_mutation.js#L60) is still only setting the `raw_text` field that we no longer use, so that should probably be changed to set the `body` field too.

Besides that case, can this field ever be `null` when fetching from Impulse otherwise?